src/Build/XMakeBuildEngine.csproj

```diff
@@ -15,32 +15,22 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.VisualStudio.Setup.Configuration.Interop" Version="1.2.304-preview5" Condition="'$(TargetFramework)' == 'net46'" />
+    <PackageReference Include="Microsoft.VisualStudio.Setup.Configuration.Interop" Version="1.2.304-preview5" Condition="'$(TargetFrameworkIdentifier)' == '.NETFramework'" />
```

Comment:
From the ref assembly it looks like we're not embedding the interop types anymore. We can add package to do this for us rather than the custom workaround target I wrote:
```XML
<PackageReference Include="Microsoft.VisualStudio.SDK.EmbedInteropTypes" Version="15.0.10" PrivateAssets="all" />
```